db/schema.rb

```diff
@@ -50,6 +50,13 @@
     t.index ["magic_link_token"], name: "index_candidates_on_magic_link_token", unique: true
   end
 
+  create_table "vendor_api_tokens", force: :cascade do |t|
+    t.string "hashed_token", null: false
+    t.datetime "created_at", precision: 6, null: false
```

Comment:
maybe adding an index since we are ordering by `created_at`?

Reply:
O wait it already has that: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/e6e104322257a7aeca0985c156db5a270298308e/db/migrate/20191002123040_create_vendor_api_tokens.rb#L5

Reply:
I've checked, it helps speed up sorting, but we can notice the difference in a large number of rows.